### ISSUE TITLE
Update not working or UI just showing wrong information? How to uninstall?

### ISSUE THREAD

**Issue by Kramerican** [netdata/netdata#5285]
UI shows I have 1.9.0-426-g64e130fa_rolling even after running the kickstarter script again

Executing `netdata -V` on command line reports netdata v1.12.0-rc3-40-nightly 

It almost looks to me that the kickstarter script installed netdata in a new location? I have netdata in /opt/netdata (old location) as well as /etc/netdata (new) now

I thought I might want to uninstall everything - but I cannot follow your uninstall instructions as I cannot find any uninstall script anywhere on the server.

What's going on? It's all really very confusing :O)

**Comment by cakrit** [comment]
We are terribly sorry about this. We had some recent changes in the installation process that have caused various issues. We are committing fixes as we see different cases on different systems.

Now, the latest kickstart.sh doesn't keep the source directory, so our instructions to uninstall using `./packaging/installer/netdata-uninstaller.sh` are incorrect for the newer versions. We will work on a fix ASAP.

To resolve your situation, we will need to remove 1.9, without losing your configuration. To give you precise instructions, it would be useful to understand how exactly you got into this situation. Are you certain that you had used the one liner kickstart.sh for the original installation of 1.9? That is, you had not used kickstart-static64.sh or the Ubuntu package manager? Also, can you please do a find for `netdata-installer.sh` so we can know if you still have the old source code in your system?

In addition, please tell us how you start netdata. The new binary should be in /usr/sbin/netdata. Perhaps you are executing a binary located in a subdirectory of /opt/netdata.

**Comment by Kramerican** [comment]
Hey there :)

No need to be sorry! Netdata is awesome and we will surely sort this out together. 

Ok so looking at my setup script for my servers, I see I have been using `bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh)` to install Netdata. I had not realized the one-liner had also changed.

I have just let the install script start netdata, and I assume it daemonizes it as it always comes up after reboot. On this particular server I tried using `systemctl start/restart netdata`

I have no particular configuration - all I have done is 

```
cat > /opt/netdata/etc/netdata/netdata.conf << EOF
[global]
 history = 86400
EOF
```

So don't worry about nuking any configuration :)

I cannot find a netdata-installer.sh file anywhere on the system ... (using `find / -name netdata-installer.sh` )

**Comment by cakrit** [comment]
Ok. you see there are two one-liners, one installs a statically binary, the other does it from source. `kickstart-static64.sh` hasn't been deprecated, but let's proceed with the easiest way to get you up and running. 

All of your old installation is under `/opt/netdata`, so you should be able to remove the entire directory, unless if the new installation with `kitckstart.sh` also put that there. Whether or not that happened, depends on `autoconf`. We can tell if any directory under `/opt' is used, after we start the correct netdata.

What we need to see is what your systemd uses. Please do a `systemctl status netdata` and show what `Loaded` says. I suspect that you have something like `/etc/systemd/system/netdata.service`, instead of `/lib/systemd/system/netdata.service`. The second one is the one that `kickstart.sh` should have put in your system after the installation and it should lead to correct path for the new netdata. If we get systemd to use the correct netdata.service, then finding which directories are used will be easy and we can clean up the rest.

**Comment by Kramerican** [comment]
Ok so running `systemctl status netdata` reveals:

`Loaded: loaded (/etc/systemd/system/netdata.service; enabled; vendor preset: enabled)`

So steps from here would be to just delete /opt/netdata ? And then I can redo my history config by doing 

```
cat > /etc/netdata/netdata.conf << EOF
[global]
 history = 86400
EOF
```

Any further steps? :O)

**Comment by cakrit** [comment]
No, we first need to tell systemd to run the correct (new) netdata. Find the new netdata.service (probably in `/lib/systemd/system/netdata.service`) and do a diff with `/etc/systemd/system/netdata.service`. You will see what I mean. I expect the ExecStart in the new one to have `/usr/sbin/netdata`, while the old one will have something under `/opt/netdata`.

Now, what goes under `/etc/systemd/system` overrides what's on the other path. So you can just delete /etc/systemd/system/netdata.service, or copy the new netdata.service over it. This will tell systems to go use the new binary. I'm not sure if you need to do a `systemctl daemon-reload` as well after the change.

When you see the new version, then we can do the cleanup of the old installation. As I've said, we need to ensure that the new netdata doesn't have anything under /opt/netdata, before we delete anything.

**Comment by Kramerican** [comment]
Howdy 

I did

```
rm /etc/systemd/system/netdata.service
cp /lib/systemd/system/netdata.service /etc/systemd/system/netdata.service
systemctl daemon-reload
systemctl restart netdata
```

And now Netdata UI frontend shows the correct version :+1: 

Next steps? :)

**Comment by cakrit** [comment]
Go to `http://yourhost:yourport/netdata.conf` and check all the paths (directories) you see there. If you see any directories under `/opt/netdata`, you want to keep them. Any other directory under `/opt/netdata` is no longer used.

**Comment by Kramerican** [comment]
As mentioned I don't need to maintain any config - also I cannot find any directories in any config statements which are not commented out :) 

In which case, I guess we are good to nuke /opt/netdata ?
